### PR TITLE
Additional disk related mappings

### PR DESCRIPTION
**Pull request by fabiand**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,20 @@
+GOFILES=$(shell find . -name \*.go)
```
Comment (rmohr) [inline Makefile]: That is not necessary. "go" does that for us. It just introduces another source for errors.

File: Makefile [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+PODFILES=$(shell find . -name \*.go -o -name Dockerfile -o -name \*.yaml -o -name \*.yaml.in -o -name \*.tpl)
```
Comment (rmohr) [inline Makefile]: I really don't like that. All manifests should be in the manifest directory. Further we hack enough around already in the "hack" folder, no need to hack here again.

File: pkg/libvirt/schema.go [hunk 3]
```diff
@@ -0,0 +1,393 @@
+package libvirt
```
Comment (rmohr) [inline pkg/libvirt/schema.go]: I guess your branch was not up to date here. This file was moved by @admiyo

File: vendor/vendor.json [hunk 4]
```diff
@@ -333,5 +333,5 @@
 			"tree": true
 		}
 	],
-	"rootPath": "kubevirt.io/kubevirt"
+	"rootPath": "kubevirt.io"
```
Comment (rmohr) [inline vendor/vendor.json]: That looks strange.

**Comment by fabiand** [comment]
Ready to review

**Comment by fabiand** [comment]
@rmohr at least I took out the makefile foo for now.

**Comment by rmohr** [comment]
retest this please

**Comment by fabiand** [comment]
@rmohr I added a few tests (and fixed some)

**Pull request merged**